sample/modules/middlewares/test-mw.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { Middleware, MiddlewareHandler } from '../../../dist';
+
+@Middleware()
+export class TestMwMiddleware implements MiddlewareHandler {
+  handler(req: Request, res: Response, next: NextFunction): void {
```

Comment:
If I'm not mistaken explicit types Request, Response and NextFunction on this line can be removed as they should be inferred from the MiddlewareHandler interface.